realsense2_camera/include/realsense2_camera/constants.h

```diff
@@ -93,7 +93,8 @@ namespace realsense2_camera
     const std::string DEFAULT_ALIGNED_DEPTH_TO_INFRA2_FRAME_ID = "camera_aligned_depth_to_infra2_frame";
     const std::string DEFAULT_ALIGNED_DEPTH_TO_FISHEYE_FRAME_ID = "camera_aligned_depth_to_fisheye_frame";
 
-    const bool DEFAULT_USE_FIX_SET_EXPOSURE = true;
+    const bool DEFAULT_USE_FIX_SET_EXPOSURE = false;  // To match ROS launch script
+    const bool DEFAULT_AUTO_EXPOSURE_SETTING = false;
```

Comment:
Too much of a correction. In ros launch the defaults are 
` <arg name="use_fix_set_auto_exposure" default="false"/>
  <arg name="auto_exposure_setting" default="true"/>`

Reply:
Oh derp! Will fix that.